server/src/test/java/org/elasticsearch/search/fetch/subphase/highlight/HighlightBuilderTests.java

```diff
@@ -717,8 +725,6 @@ private static void mutateCommonOptions(AbstractHighlighterBuilder highlightBuil
                 highlightBuilder.highlightFilter(toggleOrSet(highlightBuilder.highlightFilter()));
                 break;
             case 10:
-                highlightBuilder.forceSource(toggleOrSet(highlightBuilder.forceSource()));
-                break;
```

Comment:
with this case 10 will end up doing the same as case 11? Is that what you meant to do?